src/container.rs

```diff
@@ -513,7 +513,7 @@ impl ContainerListOptionsBuilder {
                 ContainerFilter::Label(n, v) => ("label", format!("{}={}", n, v)),
             };
 
-            param.entry(key).or_insert(Vec::new()).push(value);
+            param.entry(key).or_insert_with(Vec::new).push(value);
```

Comment:
Cool! :tada: Today I learned something because of you! :smile:

Reply:
Likewise! 🤝